react_main/src/pages/Play/CreateSetup/CreateMafiaSetup.jsx

```diff
@@ -31,22 +31,22 @@ export default function CreateMafiaSetup() {
     {
       label: "Whispers",
       ref: "whispers",
-      value: true,
+      value: false,
```

Comment:
remember to create separate PRs next time, it gets hard to track which commit made what changes, this is fine don't have to change it